packages/playground/src/pages/Wallets/Wallets.js

```diff
@@ -21,17 +21,74 @@ getOriginKey()
             // environment: 'http://localhost:8080/checkoutshopper/',
             // environment: 'https://checkoutshopper-beta.adyen.com/checkoutshopper/',
             environment: 'test',
-            onChange: handleChange,
             onSubmit: handleSubmit,
             onAdditionalDetails: handleAdditionalDetails,
             onError: console.error,
             showPayButton: true
         });
 
+        // AMAZON PAY
+        // Demo only
+        const urlSearchParams = new URLSearchParams(window.location.search);
+        const amazonCheckoutSessionId = urlSearchParams.get('amazonCheckoutSessionId');
+        const step = urlSearchParams.get('step');
+
+        // Initial state
+        if (!amazonCheckoutSessionId) {
+            window.amazonpay = checkout
+                .create('amazonpay', {
+                    currency: 'GBP',
+                    environment: 'test',
+                    merchantId: 'A3SKIS53IXYBBU',
```

Comment:
Should this go under a `configuration` object?

Reply:
Well on one hand that would align with other components, but on another we're not planning to use the `/paymentMethods` response so we wouldn't be able to get these fields from the API, so we cannot completely align it.

Reply:
I just checked and we only store the `publicKeyId` in the CA.